packages/replayio/src/utils/mixpanel/types.ts

```diff
@@ -0,0 +1,5 @@
+import { init as initMixpanel } from "mixpanel";
+
+export type MixpanelAPI = Pick<ReturnType<typeof initMixpanel>, "init" | "track">;
+
+export type EventProperties = Record<string, any>;
```

Comment:
It might be good to prefer `unknown` over `any` in cases like this. We probably don't quite *read* from this type (it's primarily positioned in write positions) for it to become a problem here though
```suggestion
export type EventProperties = Record<string, unknown>;
```

Reply:
Totally.